nevergrad/optimization/optimizerlib.py

```diff
@@ -1449,7 +1451,12 @@ class MetaModelFailure(ValueError):
     """Sometimes the optimum of the metamodel is at infinity."""
 
 
-def _learn_on_k_best(archive: utils.Archive[utils.MultiValue], k: int) -> tp.ArrayLike:
+def _train_sklearn(model, X2, y, return_dict) -> None:
+    model.fit(X2, y)
+    return_dict["model"] = model
+
+
+def _learn_on_k_best(archive: utils.Archive[utils.MultiValue], k: int, timeout: int) -> tp.ArrayLike:
```

Comment:
can we move both this functions to a different file? They are begginning to be big, and messy, in a file that is already big, and they should not serve as a model for later development...

Reply:
eg : optimization/utils.py